source/upgrade.txt

```diff
@@ -66,6 +66,10 @@ Version 6.x Breaking Changes
   ``tlsCertificateFile`` option in the ``MongoClientOptions`` type.
   Create a ``SecureContext`` object or set the ``tls``-prefixed options
   in your ``MongoClientOptions`` instance instead.
+- The driver reads files set in the ``tlsCAFile`` and
+  ``tlsCertificateKeyFile`` connection options when you first call the
```

Comment:
Suggestion:

I think "first" is redundant since I don't think you can't call it more than once.

```suggestion
  ``tlsCertificateKeyFile`` connection options when you call the
```

Applies to all instances.